plugins/oauth/hooks.js

```diff
@@ -47,7 +45,7 @@ module.exports = Object.assign({}, StoreMixin, {
     'applicationRoutes',
     'delayApplicationLoad',
     'organizationRoutes',
-    'sidebarFooter',
+    'userDropdownItems',
```

Comment:
Not part of this PR but we should use the `Mount` component instead of these filters...